packages/node_modules/@webex/plugin-meetings/src/locus-info/selfUtils.js

```diff
@@ -194,6 +221,20 @@ SelfUtils.getSipUrl = (partner, type, sipUri) => {
   return {sipUri};
 };
 
+SelfUtils.getMediaStatus = (mediaSessions = []) => {
+  const mediaStatus = {
+    audio: {},
+    video: {},
+    share: {}
+  };
+
+  mediaStatus.audio = mediaSessions.find((media) => media.mediaType === 'audio' && media.mediaContent === 'main');
```

Comment:
Media type is available as a constant in `import {eventType, trigger, mediaType} from '../metrics/config';`.
Do we have a constant for `mediaContent`?

Reply:
no will create one